src/v2/Apps/Collect/Routes/Collection/Components/Header/index.tsx

```diff
@@ -1,13 +1,12 @@
 import { ContextModule, Intent } from "@artsy/cohesion"
-import { EntityHeader, ReadMore, breakpoints } from "@artsy/palette"
+import { EntityHeader, ReadMore, Text, breakpoints } from "@artsy/palette"
 import {
   Box,
   Col,
   Flex,
   Grid,
   Row,
   Sans,
```

Comment:
Note that one instance of `Sans` remains, because it was used in the tiny artist follow component, and fixing that was leading to some rabbit-holes I didn't want to go down at this moment. Still renders as Unica.

The issue was that this wants to be small _and_ heavy, which is not an option in our current type scale. Could leave as text weight, or could break out of the scale. For now I just left it.

![Screen Shot 2020-09-21 at 4 31 55 PM](https://user-images.githubusercontent.com/140521/93818200-0cb06680-fc28-11ea-9ce3-b342b8b438e7.png)

Reply:
Updated per [Slack convo](https://artsy.slack.com/archives/C9XJKPY9W/p1600718094062700?thread_ts=1600705999.054500&cid=C9XJKPY9W)